pkg/cmd/apikey/list.go

```diff
@@ -30,11 +31,21 @@ var listCmd = &cobra.Command{
 			log.Fatal(apiclient.HandleErrorResponse(nil, err))
 		}
 
-		if output.FormatFlag != "" {
-			output.Output = apiKeyList
+		if formatFlag != "" {
+			display := output.NewOutputFormatter(apiKeyList, formatFlag)
+			display.Print()
 			return
 		}
 
 		apikey.ListApiKeys(apiKeyList)
 	},
 }
+
+func init() {
+	listCmd.PersistentFlags().StringVarP(&formatFlag, "output", "o", formatFlag, `Output format. Must be one of (yaml, json)`)
```

Comment:
Please export this as a constant from the output package: `Output format. Must be one of (yaml, json)`

If we decide to support more formats, we would then have to change only one value instead of all values in all commands.